examples/HiddenMarkovModels/SimulatedData/README.md

```diff
@@ -0,0 +1,37 @@
+# Simulated Data Example
+
+The code for this repo can be found [here](https://github.com/bonsai-rx/machinelearning-examples/tree/main/examples/HiddenMarkovModels/SimulatedData).
+
+In the following example, you can see how the Hidden Markov Model (HMM) can be used to infer the underlying hidden state of a system producing observations from 2 distinct multivariate distributions.
+
+### Dependencies
+
+If you used the bootstrapping method, you dont have to worry about the package dependencies, as these should be already installed. However, if creating a new environment or integrating into an existing one, you will need to install the following packages:
+
+* Bonsai - Core v2.8.1
```

Comment:
I wonder if we should somehow automate these listings as I imagine they can go out of date pretty quickly.

Reply:
This would be nice